apps/e2ee-server/src/utils/cryptoUtils.ts

```
@@ -0,0 +1,236 @@
+      return false;
+    }
+
+    let result = 0;
+    for (let i = 0; i < a.length; i += 1) {
+      // eslint-disable-next-line no-bitwise
+      result |= a.charCodeAt(i) ^ b.charCodeAt(i);
+    }
+
+    return result === 0;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)